recipes/rnabob/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+make clean
+make
+make install HOME=/usr/local
```

Comment:
What does HOME do? Is this the location where the binaries go? If so this should be $PREFIX isn't it?